packages/vscode-extension/package.json

```diff
@@ -60,6 +60,10 @@
       {
         "command": "neo4j.saveConnection",
         "title": "Create Connection..."
+      },
+      {
+        "command": "neo4j.switchDatabase",
+        "title": "Switch to database..."
```

Comment:
yeah, I think this is unnecessary here

it's meant to signify that an additional action must be performed to complete this action https://uxpickle.com/significance-of-the-three-dots-or-ellipses-in-ui-design/#:~:text=On%20a%20button%2C%20they%20may,the%20text%20of%20the%20sentence.

so where I've added 3 dots after 'Delete Connection...', it's because when you click that you get a modal prompt to confirm deletion

but for switching, there is no additional action.. similarly, I don't think we show the 'Create Connection...' command anywhere in the UI so it can also be removed